### DISCUSSION TITLE
Documentation system

### DISCUSSION THREAD

**Discussion by lucionmx** [supabase/supabase#3532] [Questions] [answered]
Hello,

Probably a silly question, can anyone tell me where I can find code for an open source knowledge base system just like Supabase uses for their documentation. I really like how Supabase, or Odoo have their documentation system with floating chapter menus. I-m assuming it is open source. Any help would be much appreciated.

**Comment by soedirgo** [comment, marked as answer]
Hey @lucionmx, we use https://docusaurus.io for our docs.
